frontend/src/components/eMIB/HowTo.jsx

```diff
@@ -26,20 +26,18 @@ class HowTo extends Component {
       <div>
         {this.props.inTest === false && (
           <ProgressPane
-            progressSpecs={PROGRESS_SPECS}
+            progressSpecs={SPECS}
             currentNode={0}
             paneTitle={LOCALIZE.emibTest.homePage.testTitle}
             exitButton={this.props.exitButton}
           />
         )}
-        {this.props.inTest === true && (
-          <SideNavigation progressSpecs={PROGRESS_SPECS} currentNode={0} />
-        )}
+        {this.props.inTest === true && <SideNavigation navSpecs={SPECS} currentNode={0} />}
```

Comment:
For all these, here's a cleaner way to write things:
```
{this.props.inTest && ...}
```

```
{!this.props.inTest && ...}
```

Reply:
fixed in 539ddb3